javascript-modal/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet">
+  <title>JavaScript Modal</title>
+</head>
+<body>
+<button class="open">Open Modal</button>
+<div class="modalContainer">
+<dialog class="modal">
```

Comment:
Very cool usage of the `dialog` element!! Unfortunately, the `dialog` element still isn't well-incorporated across all browsers, so you won't want to make use of it just yet. I recommend looking back to the `fixed` portion of the `css-positioning` exercise for some tips on how to accomplish this without a `dialog` element

https://caniuse.com/dialog